docs/reference/commandline/service_create.md

```diff
@@ -766,18 +766,14 @@ The options you can specify are:
 </tr>
 </thead>
 <tr>
-  <td>published and target port </td>
-  <td><tt></tt></td>
-  <td><tt></tt></td>
-  <td></td>
-</tr>
-<tr>
-  <td>protocol</td>
+  <td>published and target port</td>
   <td><tt>--publish 8080:80</tt></td>
   <td><tt>--publish published=8080,target=80</tt></td>
   <td><p>
-    The port to publish the service to on the routing mesh or directly on
-    the node, and the target port on the container.
+    The target port within the container and the port to map it to on the
+    nodes, using the routing mesh (`ingress`) or host-level networking.
```

Comment:
Looks like (because a _html_ table is used), the backticks also have to be changed to `<tt>` / `</tt>`

<img width="332" alt="screen shot 2018-01-30 at 13 44 45" src="https://user-images.githubusercontent.com/1804568/35593103-db228664-05c3-11e8-8962-7ddb9f5ab988.png">

Reply:
(Same for other occurrences in the table)

Reply:
I just forgot to set `markdown=1` on the table. I'll do that now.

Reply:
I just forgot to set `markdown=1` on the table. I'll do that now.